sdk/include/ndk/ldrfuncs.h

```diff
@@ -109,7 +109,7 @@ NTAPI
 LdrLockLoaderLock(
     _In_ ULONG Flags,
     _Out_opt_ PULONG Disposition,
-    _Out_opt_ PULONG Cookie
+    _Out_opt_ PULONG_PTR Cookie
```

Comment:
There's a lot of code that still passes an ULONG pointer to this.

Reply:
Fixed those

Reply:
I'd suggest doing those in a separate commit, but it doesn't really matter much.